tests/Nox.Types.Tests/NoxCurrencyCode3Tests.cs

```diff
@@ -0,0 +1,181 @@
+﻿namespace Nox.Types.Tests;
+public class NoxCurrencyCode3Tests
+{
+    [InlineData("AED")]
```

Comment:
Can you move this to a ClassData so we have the Test class smaller and easy to read? Thks

Reply:
Hello @rochar . Thanks for your suggestions. 

I moved this to a ClassData